From 0f47ee067d41e0291eda5484dc80d4b0a9d6f111 Mon Sep 17 00:00:00 2001
From: AngeloGioacchino Del Regno <angelogioacchino.delregno@somainline.org>
Date: Tue, 10 Aug 2021 20:42:48 +0200
Subject: [PATCH] iommu/arm-smmu-qcom: Add MSM8998 and SDM660 mss
 compatibles for identity

Add MSM8998 and SDM660's mss-pil compatibles to switch the default
iommu domain type to IDENTITY, as similarly required by SDM845 and
others.

Signed-off-by: AngeloGioacchino Del Regno <angelogioacchino.delregno@somainline.org>
(JAMI: fixup for 6.0-rc2)

Source-Commit: https://github.com/sdm660-mainline/linux/commit/0f47ee067d41e0291eda5484dc80d4b0a9d6f111
Scope: SDM660-common
Porting notes: Keep only the SDM660 MSS identity-domain match; omit the unrelated MSM8998 line from the original commit.
---
diff --git a/drivers/iommu/arm/arm-smmu/arm-smmu-qcom.c b/drivers/iommu/arm/arm-smmu/arm-smmu-qcom.c
--- a/drivers/iommu/arm/arm-smmu/arm-smmu-qcom.c
+++ b/drivers/iommu/arm/arm-smmu/arm-smmu-qcom.c
@@ -417,6 +417,7 @@
 	{ .compatible = "qcom,sc7280-mss-pil" },
 	{ .compatible = "qcom,sc8180x-mdss" },
 	{ .compatible = "qcom,sc8280xp-mdss" },
+	{ .compatible = "qcom,sdm660-mss-pil" },
 	{ .compatible = "qcom,sdm670-mdss" },
 	{ .compatible = "qcom,sdm845-mdss" },
 	{ .compatible = "qcom,sdm845-mss-pil" },
